src/hooks/useNodePairing.ts

```
@@ -0,0 +1,109 @@
+import { useCallback, useEffect, useState } from 'react'
+
+import { useMoltGateway } from '../services/molt/useMoltGateway'
+import { NodePairingResponse, PairedNode } from '../services/molt/types'
+import * as Device from 'expo-device'
+import { Platform } from 'react-native'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)